src/Components/WebAssembly/WebAssembly/src/Http/WebAssemblyHttpMessageHandler.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Http
+{
+    public static class WebAssemblyHttpMessageHandler
+    {
+        /// <summary>
+        /// Initialize a new instance of the default <see cref="HttpMessageHandler"/> required to perform HTTP operations in WebAssembly.
+        /// <para>
+        /// <see cref="HttpClient"/> instances are configured by to use the default <see cref="HttpMessageHandler"/>. This method is useful
+        /// when authoring a <see cref="DelegatingHandler"/> that performs additional work before a HTTP request is processed.
+        /// </para>
+        /// </summary>
+        /// <returns>The <see cref="HttpMessageHandler"/>.</returns>
+        public static HttpMessageHandler CreateDefault()
```

Comment:
We could make this a static readonly field. The implementation already assumes [it's a singleton](https://github.com/mono/mono/blob/master/sdks/wasm/framework/src/WebAssembly.Net.Http/WasmHttpMessageHandler.cs#L12). But I don't see this API being called multiple-times over.